protocol/app/app.go

```diff
@@ -613,7 +610,10 @@ func New(
 		if daemonFlags.Liquidation.Enabled {
 			app.LiquidationsClient = liquidationclient.NewClient(logger)
 			go func() {
-				app.MonitorDaemon(app.LiquidationsClient, MaximumDaemonUnhealthyDuration)
+				app.MonitorDaemon(
+					app.LiquidationsClient,
+					time.Duration(daemonFlags.Shared.MaxDaemonUnhealthySeconds)*time.Second,
```

Comment:
nit: given that this is common across all daemons, you could calc this once and reuse.